app/api/orders.py

```diff
@@ -34,7 +34,7 @@ class OrdersListPost(ResourceList):
     """
     def before_post(self, args, kwargs, data=None):
         """
-        before post method to check for required relationship and proper permission
+        before post method to check for required relationships and proper permission
```

Comment:
"before post method to check for required relationships permissions"

Reply:
Why ? We are doing two things here:
1. Checking if the required relationships are present in the data or not. (event and attendees)
2. If we are the event organizer or not.

@mayank8318 Does this clear things up for you ?

Reply:
My bad I meant this "before post method to check for required relationships and permissions"